source/MaterialXView/Viewer.cpp

```diff
@@ -209,6 +210,10 @@ Viewer::Viewer(const std::string& materialFilename,
     _searchPath(searchPath),
     _libraryFolders(libraryFolders),
     _meshScale(1.0f),
+    _turntableEnabled(false),
+    _turntableSteps(360),
+    _turntableStep(0),
+    _turntableRotation(0.0f),
```

Comment:
@kwokcb This might be clearer if we removed the shadow variable `_turntableRotation`, computing it as needed from `_turntableStep`, rather than maintaining both variables and keeping them in sync.

Reply:
Makes sense. Removed now.